pkg/internal/checkup/trex/client.go

```diff
@@ -151,6 +151,12 @@ func (t Client) ClearStats(vmiName string) (string, error) {
 	return t.runTrexConsoleCmd(vmiName, "clear")
 }
 
+func (t Client) StartTraffic(vmiName string, port PortIdx) (string, error) {
+	testDurationSeconds := int(t.testDuration.Seconds())
+	return t.runTrexConsoleCmd(vmiName, fmt.Sprintf("start -f %s/testpmd.py -m %spps -p %d -d %d",
```

Comment:
Please consider building the command in stages using a string builder.

Reply:
DONE